src/Microsoft.PowerShell.MarkdownRender/VT100EscapeSequences.cs

```diff
@@ -79,16 +80,48 @@ public sealed class MarkdownOptionInfo
         /// <returns>Specified property name as escape sequence.</returns>
         public string AsEscapeSequence(string propertyName)
         {
-            var propertyValue = this.GetType().GetProperty(propertyName)?.GetValue(this) as string;
+            var propName = propertyName?.ToLower();
```

Comment:
I don't like this. Given that this method is public can we use enum type parameter like
string propertyName -> EscapeSequenceEnum propertyName ?

Reply:
Changed it to an enum